template_content/.algokit.toml.jinja

```diff
@@ -0,0 +1,16 @@
+[algokit]
+min_version = "v1.1.0-beta.4"
+
+{%- if deployment_language == 'python' %}
+[deploy.localnet]
+command = "poetry run python -m smart_contracts deploy"
+
+[deploy.testnet]
+command = "poetry run python -m smart_contracts deploy"
+{%- elif deployment_language == 'typescript' %}
+[deploy.localnet]
```

Comment:
just a side note since this is branched out from deploy branch - i'll make sure to propagate the deploy changes first then otherwise we might want to cherry pick the deploy specific changes out (which should be relatively easy to do).